assets/scss/custom.scss

```diff
@@ -288,16 +288,20 @@ footer {
 /* Styles for sections that are rendered from data, such as HTTP APIs and event schemas */
 .rendered-data {
   background-color: $secondary-lightest-background;
-  padding: 1rem;
-  margin: 1rem 0;
+  padding: 0.85rem;
+  margin: 0.85rem 0;
 
   details {
     summary {
-      padding: .5rem 0;
-      p {
-        max-width: 80%;
+      h1 {
+        margin: 0;
+        /* Ensure the disclosure control is vertically centred with the header text. */
+        vertical-align: middle;
       }
```

Comment:
I guess this relies on the first line of the summary being an `<h1>`, but that's probably an assumption we can live with.

Reply:
Could maybe do `summary > *` or `summary > :first-child` if this bites us in the future/